tests/test_middleware.py

```diff
@@ -2,7 +2,7 @@
 from django.http import HttpResponse
 
 import mock
-from mohawk.exc import MacMismatch
+from mohawk.exc import MisComputedContentHash
```

Comment:
> ok, this needs to change to mohawk>=0.3.3

The reason I didn't update it, is since it's only the tests (which are possibly testing things outside of hawkrest's remit IMO) that fail with the newer version of mohawk. hawkrest itself is still compatible with the older mohawk version.

Happy to update it regardless though, if you'd prefer.